backend/app/adapter/gqlapi/resolver/authquery.go

```diff
@@ -55,6 +55,26 @@ func (v AuthQuery) ChangeLog() (ChangeLog, error) {
 	return newChangeLog(changeLog, lastViewedAt), err
 }
 
+// Changes retrieves all the changes that exists in the persistent storage
```

Comment:
```suggestion
// Changes retrieves all the changes that exists in the persistent storage.
```
nit: The comments in go should end with a full stop.